azure_ad_admin/help.md

```diff
@@ -34,13 +34,6 @@ The connection configuration accepts the following parameters:
 Example input:
 
 ```
-{
```

Comment:
Did this example input get blown out accidentally

Reply:
Yup, fixed.